### DISCUSSION TITLE
Are Blazored Controls Compatible with ASP.NET Core 7.0 Blazor

### DISCUSSION THREAD

**Discussion by Orgbrat** [Blazored/Modal#482] [Q&A]
I have ASP.NET Core project that needs to be migrated from .NET 6.0 to.NET 7.0 and it is using some Blazored controls.  Are the Blazored controls compatible with .NET Core 7.0? If not when do you think they will be ready for Blazor 7.0?

**Comment by ErikGrav** [comment]
I seem to be facing this problem when using NET7: 
https://github.com/Blazored/Modal/issues/454
Modal windows pops up at the top of the page. 
@chrissainty 
I will try to made a sample. 

Edit: 

The changes to CSS might cause the issue..

**Reply by LoupaLoupa** [reply]
I had the same issue and I was able to fix it by using !important to the display: flex in the css class we pass to the modal when it it created using options.PositionCustomClass = admin-modal-overlay

where

.admin-modal-overlay {
    display: flex !important;
    z-index: 1300;
    align-items: center;
    justify-content: center;
}
